cache/sampleStore.js

```diff
@@ -0,0 +1,103 @@
+/**
+ * Copyright (c) 2017, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or
+ * https://opensource.org/licenses/BSD-3-Clause
+ */
+
+/**
+ * ./cache/sampleStore.js
+ *
+ * Redis Cache
+ */
+'use strict'; // eslint-disable-line strict
+const PFX = 'samsto';
+const SEP = ':';
+const constants = {
+  featureName: 'enableRedisSampleStore',
+  fieldsToStringify: {
+    aspect: [
+      'relatedLinks',
+      'tags',
+      'criticalRange',
+      'warningRange',
+      'infoRange',
+      'okRange',
+    ],
+    sample: ['relatedLinks'],
+  },
+  indexKey: {
+    aspect: PFX + SEP + 'aspects',
+    sample: PFX + SEP + 'samples',
```

Comment:
I thought all the /aspects endpoints are going to hit this redis aspect store. So this will also act like the normal API cache.  
Not sure which way to go here.